Assets/HoloToolkit/Utilities/Scripts/Singleton.cs

```diff
@@ -11,7 +11,8 @@ namespace HoloToolkit.Unity
     /// parent root GameObject with <see cref="Object.DontDestroyOnLoad"/></remarks>
     /// </summary>
     /// <typeparam name="T">The Singleton Type</typeparam>
-    public class Singleton<T> : MonoBehaviour where T : Singleton<T>
+    [DisallowMultipleComponent]
+    public abstract class Singleton<T> : MonoBehaviour where T : Singleton<T>
```

Comment:
I think this change is outside the scope of this PR and should be addressed separately.